64728: Update pool available space description (#651)
freenas/freenas-docs: userguide/storage.rst

in :menuselection:`Storage --> Pools`.

In the example shown in :numref:`Figure %s <zfs_vol_fig>`, the created
pool is named *pool1*. The description indicates that it is healthy and
shows how much space is :guilabel:`Used` and :guilabel:`Free`.

Click the down arrow to see more details about the pool. This second
entry has the same name and represents the implicit or root dataset. The

> :guilabel:`Used` and :guilabel:`Available` entries reflect the pool

Reviewer: ```suggestion
:guilabel:`Used` and :guilabel:`Available` entries show the amount
```

> :guilabel:`Used` and :guilabel:`Free` space. The entry also indicates

Reviewer: ```suggestion
of space used and available. Also shown are
```
